### ISSUE TITLE
table relation missing in postgres

### ISSUE THREAD

**Issue by johnsonbrothers** [Admidio/admidio#1529]
**Describe the bug**

database error in postgres log when starting a new installation of admidio 4.2.14 using container

2023-11-27 10:45:40.335 UTC [174] ERROR:  relation "adm_organizations" does not exist at character 20
2023-11-27 10:45:40.335 UTC [174] STATEMENT:  SELECT org_id FROM adm_organizations

**To Reproduce**
i'm using podman to run containers
1. start postgres container
2. create database in postgres
3. start admidio container
4. go to start webbpage to start configure admidio
5. connect to database
6. See error in postgres logfile

**Expected behavior**

No error should show up in postgres logfile

**Screenshots**

**System (please complete the following information):**
 - Admidio-Version container Admidio tag latest i.e. 4.2.14
 - PHP n/a, using container
 - Database PostgreSQL
 - Browser firefox
 - Browser-Version 119.0.1

**Additional context**

**Comment by Fasse** [comment]
Does the installation abort at this point or is it "only" the log entry that you report?

**Comment by johnsonbrothers** [comment]
Sorry,

The installation continues and I can log in to admidio, but I dont know if Im missing functionality in the system.

**Comment by johnsonbrothers** [comment]
Oh, okay, I see.
You could check if the table exist instead of expecting an error :-)

**Comment by Fasse** [comment]
Yes your are right. I thought about this. But than the implementation would be database specific. Because every database has it's own syntax for checking existing tables.
